README.md

```diff
@@ -15,16 +15,21 @@ To run the demo notebooks yourself:
 
 ## Documentation
 
-The demo notebooks in this repository serve as supplementary documentation for the AutoMLx package. The AutoMLx class documentation is available on [docs.oracle.com](https://docs.oracle.com/en-us/iaas/tools/automlx/latest/html/multiversion/v23.1.1/index.html).
+The demo notebooks in this repository serve as supplementary documentation for the AutoMLx package. The AutoMLx class documentation is available on [docs.oracle.com](https://docs.oracle.com/en-us/iaas/tools/automlx/latest/v24.3.0/index.html).
```

Comment:
You can link to https://docs.oracle.com/en-us/iaas/tools/automlx/latest/latest/ instead